src/common/axes/AxisTicks.jsx

```diff
@@ -221,7 +221,11 @@ module.exports = React.createClass({
               textAnchor={textAnchor}
               {...optionalTextProps}
             >
-              {tickFormat(tick)}
+              {`${tickFormat(tick)}`.split('\n').map((tickLabel) => (
```

Comment:
Thats what i did earlier but some tests failed with that so was forced into using template string
```
PhantomJS 2.1.1 (Mac OS X 0.0.0) BarChart renders barchart FAILED
	undefined is not a constructor (evaluating 'tickFormat(tick).split('\n')')
```
and 2 other failures